eslint/babel-eslint-plugin/src/rules/valid-typeof.js

```diff
@@ -1,8 +0,0 @@
-import ruleComposer from "eslint-rule-composer";
-import eslint from "eslint";
-
-const validTypeOf = new eslint.Linter().getRules().get("valid-typeof");
-
-export default ruleComposer.filterReports(validTypeOf, problem => {
-  return problem.node.value !== "bigint";
```

Comment:
This isn't fixed yet, but will be coming to ESLint core once https://github.com/estree/estree/pull/204 is merged.